ontology/coap.html

```diff
@@ -0,0 +1,52 @@
+
+<!DOCTYPE html>
+<html>
+    <head>
+    <meta charset='utf-8'>
+    <script src='https://www.w3.org/Tools/respec/respec-w3c-common' class='remove'></script>
+    <script class='remove'>
+        var respecConfig = {
+            specStatus: "ED",
+            editors: [{name:"Victor Charpenay",company:"Siemens AG",companyURL:"https://www.siemens.com/"},{name:"Ege Korkan",company:"Siemens AG",companyURL:"https://www.siemens.com/"},{name:"Sebastian Käbisch",company:"Siemens AG",companyURL:"https://www.siemens.com/"},{name:"Michael Koster",company:"Invited Expert",companyURL:"https://github.com/mjkoster/"},{name:"Jan Romann",company:"Invited Expert",companyURL:"https://github.com/JKRhb/"}],
```

Comment:
I am not sure that we should use this list of authors. I think that the editors of the CoAP binding should be the authors too.

Reply:
Very good point, for now I simply copied over the old author and contributer list. Should the currently listed contributors be kept? Or should we start from scratch here?

Reply:
I would say from scratch. You should add your name though. I have talked with @ektrah and he is ok to make you another author of the coap binding but we will discuss this separately at https://www.w3.org/WoT/IG/wiki/WG_WoT_Thing_Description_WebConf#February_22.2C_2023

Reply:
Thank you for the feedback! I started to adjust the author lists in 1a82bc5bcb75a7eb6204b91036edf24194c0ed4f